PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/webapp/webappconfig/RiderWebAppConfigurationType.kt

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.intellij.runner.webapp.webappconfig
+
+import com.intellij.execution.configurations.ConfigurationFactory
+import com.intellij.execution.configurations.ConfigurationType
+import icons.RiderIcons
+import org.jetbrains.annotations.Nls
+import javax.swing.Icon
+
+class RiderWebAppConfigurationType : ConfigurationType {
+
+    companion object {
+
+        private const val RUN_CONFIG_TYPE_ID = "RiderAzurePublish"
```

Comment:
Let's clarify in ID that it's `Web` publish because we're going to have DB publish also

Reply:
Done.